### PR TITLE
Sync from mbedos

### PR DESCRIPTION
**Pull request by artokin**
## Status
**READY**

## Migrations
NO

## Description
Sync changes from Mbed OS back to master repo to make future maintenance easier

### CODE REVIEW

File: source/ns_cmdline.c [hunk 1]
```diff
@@ -215,7 +215,7 @@
 #endif
 // Maximum length of input line
 #ifndef MBED_CONF_CMDLINE_MAX_LINE_LENGTH
-#define MBED_CONF_CMDLINE_MAX_LINE_LENGTH 2000
+#define MBED_CONF_CMDLINE_MAX_LINE_LENGTH 200
```
Comment (teetak01) [inline source/ns_cmdline.c:218]: why this default change?

Comment (artokin) [reply source/ns_cmdline.c:218]: To optimise RAM consumption: https://github.com/ARMmbed/mbed-os/issues/13438

Comment (teetak01) [reply source/ns_cmdline.c:218]: Ok, well its probably a bit of a breaking change. At least for our testing :-)

Comment (teetak01) [reply source/ns_cmdline.c:218]: well, not a big deal. Easy to configure it back to larger value for testing.

**Pull request merged**